ci/eval/default.nix

```diff
@@ -66,6 +67,9 @@ let
             "$src/ci/eval/attrpaths.nix" \
             -A paths \
             -I "$src" \
+            --arg extraNixpkgsConfig "${
+              lib.generators.toPretty { multiline = false; } extraNixpkgsConfig
+            }" \
```

Comment:
I'd feel more comfortable passing around a JSON string.

`toPretty` is great for printing nix values, but it doesn't make any guarantees that the printed value will be 100% accurate. It's more intended for printing values in error messages and debugging.

If we're assuming the extra config is something that can be represented as plain data (which it should be), then we can be explicit about that and use `toJSON`, `fromJSON`, and `--argstr`.

Reply:
Roughly :+1: See my followup comments.